scripts/test_usb_scan.py

```diff
@@ -143,7 +143,8 @@ def test_usb_exit(self, devices, interfaces, results,
         with self.assertRaises(SystemExit) as cm:
             self.test_usb_common(devices, interfaces, results, path)
         if msg:
-            self.assertIn(msg, cm.exception.code)
+            # cm.exception.code is int type whose format like "duplicated tag'vid' found, malformed line ALLOW:vid=056a vid=0314 class=03"
```

Comment:
I think it would be better to resolve this pylint warning.

Reply:
Unresolved this discussion because it was set to resolved without a comment,
and it was not resolved (fixed) but disabled.

Reply:
I have a comments there that is an explain why disable pytype error, pytype think `cm.exception.code` is int, but actually its output is a str.